itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcidTables.java

```diff
@@ -592,7 +577,8 @@ public void testFailoverDuringDumpWithPreviousFailed() throws Throwable {
     fs.delete(dumpAckFile, false);
 
     dumpData = primary.run("use " + primaryDbName)
-            .run("insert into t2 partition(name='Carl') values(10)")
+            .run("create table t3 (id int)")
+            .run("insert into t3 values (2)")
```

Comment:
insert into existing table as well. The table which has already part of repl and assert that the change is not there